Packs/Base/Scripts/script-CommonServer.yml

```
@@ -2097,7 +2097,77 @@ script: >-
 
     /**************************** REGEX FORMATTING end *******************************/
 
+  function getDemistoVersion() {
+      try {
+          return demistoVersion();
+      } catch (ex) {
+          throw 'Failed retriving server version - ' + ex;
+      }
+  }
+
+  function isDemistoVersionGE(version, buildNumber) {
+      var serverVersion = {};
+      try {
+          serverVersion = getDemistoVersion();
+          if (serverVersion.version > version) {
+              return true;
+          } else if (serverVersion.version === version) {
+              if (buildNumber !== undefined) {
+                  var intBuildNumber = parseInt(serverVersion.buildNumber);
+                  if (isNaN(intBuildNumber)) {
+                      // dev editions are not comparable
+                      logDebug('isDemistoVersionGE: Error.\ninput: server version: {0} build number: {1}\nserver version: {2}'.format(version, buildNumber, JSON.stringify(serverVersion)));
```

Better to log everything on one line. 
```suggestion
                      logDebug('isDemistoVersionGE: Error. Input: version: [{0}] buildNumber: [{1}]. server version: {2}'.format(version, buildNumber, JSON.stringify(serverVersion)));
```